eldritch/monsters.py

```diff
@@ -427,6 +427,29 @@ def Zombie():
   )
 
 
+class EventMonster(Monster):
+  """A pseudo-monster used for events like Bank3 and Hospital2"""
+
+  def __init__(self, name, rating, pass_event, fail_event, toughness=1, attributes=None):
+    super().__init__(
+        name, "normal", "moon", {"evade": 0} | rating,
```

Comment:
Using `|` on dictionaries is a feature introduced in python 3.9. To preserve 3.6 compatibility (actually, I should update everything to use 3.7 since 3.6 has been end of lifed), you can use something like `dict(evade=0, **rating)`.